src/insert.rs

```diff
@@ -256,6 +259,12 @@ impl<T> Insert<T> {
         self.state.terminated();
         self.wait_handle().await
     }
+    
+    /// Returns the number of serialized bytes that have been written because of
+    /// write operations.
+    pub fn written_bytes(&self) -> usize {
```

Comment:
Also, it's questionable to use `usize` vs `u64` here.

Formally, it's possible to write more than 4GiB on 32b systems inside one `INSERT` query.

So, I think we should use `u64` here.